src/chain-definition-custom/chain_def_testnet_poa_v0.1.0.json

```diff
@@ -2,7 +2,7 @@
   "name": "DataHighway Testnet",
   "id": "testnet-latest",
   "bootNodes": [
-    "/ip4/127.0.0.1/tcp/30333/p2p/QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ"
+    "/ip4/172.31.1.212/tcp/30333/p2p/QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ"
```

Comment:
the bootnodes should be changed in src/chain_spec.rs.
this file is only updated based on changes to chain_spec_testnet_latest.json when you run the following commands after building the codebase (see steps in the README):
```
./target/release/datahighway build-spec \
  --chain=testnet-latest > ./src/chain-spec-templates/chain_spec_testnet_latest.json

./target/release/datahighway build-spec \
  --chain ./src/chain-spec-templates/chain_spec_testnet_latest.json \
  --raw > ./src/chain-definition-custom/chain_def_testnet_v0.1.0.json
```

Reply:
i.e. the change should be made to this line https://github.com/DataHighway-DHX/node/blob/master/src/chain_spec.rs#L227, and it should be changed to `"/ip4/172.31.1.212/tcp/30333/p2p/QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ".to_string(),`

Reply:
Thanks, I think it's better not to set this IP in that file at all actually and instead use the service names from docker-compose file (alice/bob). If the containers are started thru docker-compose it will resolve ok by docker-compose process and links. What do you think?

Reply:
agreed